src/core/system.h

```diff
@@ -37,7 +37,7 @@ class System {
      * @param system_id System id.
      * @param comp_id Component id.
      */
-    explicit System(MavsdkImpl &parent, uint8_t system_id, uint8_t comp_id);
+    explicit System(MavsdkImpl &parent, uint8_t system_id, uint8_t comp_id, bool connected);
```

Comment:
The param `connected` is not explained in the docs above.

I don't understand what it exactly means and the problem is that "connected" is an overloaded term in the MAVSDK/MAVLink world already, so I'd be cautious in using it. Please explain what it entails and then maybe we can come up with something better.

Reply:
Added explanation.